src/main/java/com/itrex/java/lab/service/impl/FillingDatabaseTestServiceImpl.java

```diff
@@ -0,0 +1,133 @@
+package com.itrex.java.lab.service.impl;
+
+import com.itrex.java.lab.entity.Contract;
+import com.itrex.java.lab.entity.Offer;
+import com.itrex.java.lab.entity.Role;
+import com.itrex.java.lab.entity.Status;
+import com.itrex.java.lab.entity.User;
+import com.itrex.java.lab.repository.data.ContractRepository;
+import com.itrex.java.lab.repository.data.OfferRepository;
+import com.itrex.java.lab.repository.data.UserRepository;
+import com.itrex.java.lab.service.FillingDatabaseTestService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+@Service
+@AllArgsConstructor
+public class FillingDatabaseTestServiceImpl implements FillingDatabaseTestService {
+
+    private static int userNumber = 1;
+    private static final String BASE_CUSTOMER_NAME = "customer_generated";
+    private static final String BASE_CONTRACTOR_NAME = "contractor_generated";
+    private static final int MIN_CONTRACT_AMOUNT = 5;
+    private static final int MAX_CONTRACT_AMOUNT = 20;
+    private static final int MIN_OFFER_AMOUNT = 5;
+    private static final int MAX_OFFER_AMOUNT = 15;
+
+    private static final Random RANDOM_GENERATOR = new Random();
+
+    private final UserRepository userRepository;
+    private final ContractRepository contractRepository;
+    private final OfferRepository offerRepository;
+
+    @Override
+    public boolean fillUserTable(String role, int quantity) {
+        if (Role.CUSTOMER.name().equals(role)) {
+            List<User> userList = new ArrayList<>();
+            for (int i = 0; i < quantity; i++) {
+                User user = User.builder()
+                        .name(BASE_CUSTOMER_NAME + userNumber)
+                        .password("$2a$12$ma0C6gCWmZL/d3FSt78mF.FCcLdfBqc4XXyRrDBv54EbWEPcj3OIC")
+                        .role(Role.CUSTOMER)
+                        .status(Status.ACTIVE)
+                        .email(BASE_CUSTOMER_NAME + userNumber + "@gmail.com")
```

Comment:
Use String.format(...) for contactionation
Move all contants to static final variables

Reply:
Done.